src/main/java/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+package dao;
+
+import exceptions.DataProcessingException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import models.Manufacturer;
+import util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String request = "INSERT INTO manufacturers(name, country) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement = connection

```

Comment:
You can call PreparedStatement and query string objects just statement and query because we already know their context by the name of the method 
```suggestion
                PreparedStatement statement = connection

```